### ISSUE TITLE
The contents of document is old

### ISSUE THREAD

**Issue by JunkoKawazoe** [MicrosoftDocs/SupportArticles-docs#710]
[Enter feedback here]
I think that these functions are integrated.

●WindowsAzureNetAgent.exe: Windows Azure Network Agent service
●WindowsAzureTelemetryService.exe: Windows Azure Telemetry Service

So, it is not appropriate to ask readers of this document to confirm these functions in process.
It is possible that this description can make serious readers confuse.
Please fix it to be correct content.

---
#### Document Details
https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/windows-azure-guest-agent#troubleshooting-vm-agent-that-is-in-not-ready-status

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 95825442-ec08-fcdd-76bd-cc0694b6b913
* Version Independent ID: 11a97acc-d077-3bac-9411-8199bd98b255
* Content: [Troubleshooting Azure Windows VM Agent - Virtual Machines](https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/windows-azure-guest-agent#troubleshooting-vm-agent-that-is-in-not-ready-status)
* Content Source: [support/azure/virtual-machines/windows-azure-guest-agent.md](https://github.com/MicrosoftDocs/SupportArticles-docs/blob/main/support/azure/virtual-machines/windows-azure-guest-agent.md)
* Service: **virtual-machines**
* GitHub Login: @dkdiksha
* Microsoft Alias: **genli**

**Comment by genlin** [comment]
@JunkoKawazoe, Thanks for the feedback. I will investigate and update the article.

**Comment by genlin** [comment]
@JunkoKawazoe, I removed the steps about WindowsAzureNetAgent.exe from the article.  Some old VMs may still have Windows Azure Telemetry Service, so I would keep it.  I added the following note to avoid confusion:

Starting in version 2.7.41491.971 of the VM Guest Agent, the Telemetry component is included in the RD Agent service, Therefore, you might not see this Telemetry service listed in newly created VMs.

I will process to close this issue. Feel free to let me know if there is any additional question or comment.
